Description: Check for the "root" group also on non-linux architectures.
Author: Felix Geyer <fgeyer@debian.org>
Forwarded: yes

--- a/tests/daemon_tests.c
+++ b/tests/daemon_tests.c
@@ -34,7 +34,7 @@ END_TEST
 
 START_TEST(getgroup_root_string)
 {
-#if defined(__linux__)
+#if defined(__linux__) || defined(__GNU__) || defined(__GLIBC__)
 	ck_assert_int_eq((int)getgroup("root"), 0);
 #else
 	ck_assert_int_eq((int)getgroup("wheel"), 0);
